### DISCUSSION TITLE
Rounding mutation feature (.net)

### DISCUSSION THREAD

**Discussion by aralmo** [stryker-mutator/stryker-net#2067] [Ideas]
The idea would be to mutate used rounding type if math.round is used in a method. 
I've seen pretty nasty errors due to the fact that it uses bankers rounding by default, wich may be counter intuitive from a class called Math.

**Comment by richardwerkman** [comment]
We have an open issue for this, you could join the discussion there and maybe build the mutator if you feel like it? See #1194
